<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_prod</class>
 <widget class="QDialog" name="edit_prod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_name_edit">
       <property name="text">
        <string>Изменить название:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Name_edit_prod"/>
     </item>
     <item>
      <widget class="QLabel" name="label_art_edit">
       <property name="text">
        <string>Изменить артикул:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Art_edit_prod"/>
     </item>
     <item>
      <widget class="QLabel" name="label_price_edit">
       <property name="text">
        <string>Изменить цену:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="price_edit_prod"/>
     </item>
     <item>
      <widget class="QLabel" name="label_count_edit">
       <property name="text">
        <string>Изменить количество товаров:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="edit_prod_button">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Изменить продукт</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
